packages/api/src/routes/lightclient.ts

```diff
@@ -1,52 +1,54 @@
-import {Path} from "@chainsafe/ssz";
+import {ContainerType, Path, VectorType} from "@chainsafe/ssz";
 import {Proof} from "@chainsafe/persistent-merkle-tree";
-import {altair, ssz, SyncPeriod} from "@chainsafe/lodestar-types";
-import {
-  ArrayOf,
-  reqEmpty,
-  ReturnTypes,
-  RoutesData,
-  Schema,
-  sameType,
-  ContainerData,
-  ReqSerializers,
-  ReqEmpty,
-} from "../utils";
+import {altair, phase0, ssz, SyncPeriod} from "@chainsafe/lodestar-types";
+import {ArrayOf, ReturnTypes, RoutesData, Schema, sameType, ContainerData, ReqSerializers} from "../utils";
 
 // See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
 
+export type LightclientSnapshotWithProof = {
+  header: phase0.BeaconBlockHeader;
+  currentSyncCommittee: altair.SyncCommittee;
+  // TODO: Not really necessary since it can be fetched with an update
+  nextSyncCommittee: altair.SyncCommittee;
+  /** Single branch proof from state root to currentSyncCommittee parent */
+  syncCommitteesBranch: Uint8Array[];
+};
+
 export type Api = {
-  /** TODO: description */
+  /**
+   * Returns a multiproof of `paths` at the requested `stateId`.
+   * The requested `stateId` may not be available. Regular nodes only keep recent states in memory.
+   */
   getStateProof(stateId: string, paths: Path[]): Promise<{data: Proof}>;
-  /** TODO: description */
-  getBestUpdates(from: SyncPeriod, to: SyncPeriod): Promise<{data: altair.LightClientUpdate[]}>;
-  /** TODO: description */
-  getLatestUpdateFinalized(): Promise<{data: altair.LightClientUpdate}>;
-  /** TODO: description */
-  getLatestUpdateNonFinalized(): Promise<{data: altair.LightClientUpdate}>;
   /**
-   * Fetch a proof needed for light client initialization
+   * Returns an array of best updates in the requested periods within the inclusive range `from` - `to`.
+   * Best is defined by (in order of priority):
+   * - Is finalized update
+   * - Has most bits
+   * - Oldest update
```

Comment:
Shouldn't newer update (i.e., later slot) be preferred over older one, to delay landing in the fallback branch of `process_light_client_update` after initial sync? https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md

Reply:
This implementation of the server diverges from the spec's proposal in that it has two types of updates:
- committee_update: You want to oldest to minimize the chance or re-org
- header_update: You want to newer / latest